GearRent/Networking/APIManager.h

```diff
@@ -14,6 +14,7 @@ NS_ASSUME_NONNULL_BEGIN
 @interface APIManager : NSObject
 
 void fetchListingsWithCoordinates(NSArray<CLLocation *> *coordinates, void(^completion)(NSArray<Item *> *, NSError *error));
+void fetchNearestCity(double lat, double longitude,void(^completion)(NSString *, NSError *error));
```

Comment:
using CGFloat instead of double is more common; add a space before the void

Reply:
It might be better to just pass a `CLLocation` so you have less parameters.

Reply:
I'm getting errors when I try to put the new completions in the function definition?